src/include/daos/btree.h

```diff
@@ -528,7 +528,7 @@ int  dbtree_open_inplace_ex(struct btr_root *root, struct umem_attr *uma,
 			    daos_handle_t coh, void *priv, daos_handle_t *toh);
 int  dbtree_close(daos_handle_t toh);
 int  dbtree_destroy(daos_handle_t toh, void *args);
-int  dbtree_drain(daos_handle_t toh, int *credits, void *args, bool *destroyed);
+int  dbtree_drain(daos_handle_t toh, int *credits, void *args, bool *destroy);
```

Comment:
'destroyed' looks more proper to me.

Reply:
It's due to overloading the argument.  It now controls whether or not the table is to be destroyed or not.

I think it may be a little clear with either two APIs or two arguments, so somethinglike

int dbtree_drain(daos_handle toh, int *credits, void *args, bool destroy_on_empty, bool *is_empty)